ios/Classes/SwiftTwilioVoicePlugin.swift

```diff
@@ -154,6 +154,11 @@ public class SwiftTwilioVoicePlugin: NSObject, FlutterPlugin,  FlutterStreamHand
             }
             eventSink(speakerIsOn ? "Speaker On" : "Speaker Off")
         }
+        else if flutterCall.method == "call-sid"
+        {
+            result(self.call!.sid);
```

Comment:
thought: I don't like the idea about force unwrapping, I think it can be null if there's no active call

Reply:
right, sorry - I should have added that too. Thanks

Reply:
Hey, I think you still missed this :)